pytest_mock.py

```diff
@@ -254,6 +260,7 @@ def wrap_assert_methods(config):
         'assert_not_called': wrap_assert_not_called,
         'assert_called_with': wrap_assert_called_with,
         'assert_called_once_with': wrap_assert_called_once_with,
+        'assert_called_once': wrap_assert_called_once,
```

Comment:
You probably need to add this only in Python 3.6.

Reply:
It's definitely present in [Mock on PyPI](https://pypi.python.org/pypi/mock) too. I actually noticed it on Python 2.7 because `mocker.spy()` didn't have that method... Is there a nice way to tell if we're using the stdlib or external mock?

Reply:
Ahh good point. But I just noticed that what actually fails is the *test*, so you can change your test thus:

```python
    def test_assert_called_once_wrapper(mocker):
        stub = mocker.stub()
        if not hasattr(stub, 'assert_called_once'):
            pytest.skip('assert_called_once not available')
        stub("foo")
        ...
```